ha/core/system_health/status_mapper.py

```diff
@@ -27,15 +30,20 @@ class StatusMapper:
         "fault_resolved": "online",
         "missing": "offline",
         "insertion": "online",
+        "recovering": "recovering",
         "online": "online",
         "failed": "failed",
         "unknown": "unknown",
+        "degraded": "degraded",
         "threshold_breached:low": "degraded",
         "threshold_breached:high": "degraded"
     }
 
     EVENT_TO_SEVERITY_MAPPING = {
+        "starting": "informational",
+        "recovering": "informational",
         "online": "informational",
+        "degraded": "informational",
```

Comment:
degraded can also be mapped to informational?

Reply:
I guess so. You have some different suggestion?

Reply:
I thought error. But I guess No.. This is fine